deploy/docker-compose.yml

```diff
@@ -78,15 +78,42 @@ services:
         fluentd-address: "0.0.0.0:24224"
         tag: f5-do
 
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.2.0
+    container_name: F5-ADCAAS-ELASTICSEARCH
+    ports:
+      - 9200:9200
+    expose:
+      - 9200
+    environment:
+      - discovery.type=single-node
+
   FLUENTD:
-    image: fluent/fluentd:latest
+    build: ./docker/fluent
+    image: f5-adcaas-fluent
     container_name: FLUENTD
     ports:
       - 24224:24224
+      - 24224:24224/udp
     volumes:
       - ./fluentd.conf:/fluentd/etc/test.conf
+    depends_on:
+      - elasticsearch
+      - kibana
     environment:
       - FLUENTD_CONF=test.conf
+
+  kibana:
+    image: docker.elastic.co/kibana/kibana:7.2.0
+    container_name: F5-ADCAAS-KIBANA
+    depends_on:
+      - elasticsearch
+    ports:
+      - 5601:5601
+    expose:
+      - 5601
+    volumes:
```

Comment:
We may need to mount kibana data directory as a volume so that user-created index data can be persisted.

Reply:
Yes, we can do that, we can have a investigate in next sprint about HA of Kibana, Elasticserach, Fluentd.